src/electionguard/data_store.py

```diff
@@ -12,18 +12,18 @@
 )
 
 
-T = TypeVar("T")
+_T = TypeVar("_T")
 U = TypeVar("U")
```

Comment:
Love this PR. Would you mind also doing this for `U` here in this file?

Reply:
Sure no problem. Made the changes and squashed it with the previous commit. Additionally made U private in one other file  utils.py for consistency.